deployer/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM deployer
 
 COPY src/deployer.sh /bin/deployer
-RUN chmod +x /bin/deployer
+COPY docker/post-to-slack.sh /bin/post-to-slack
```

Comment:
post-to-slack used to be automatically injected into every image, but only etna and deployer images needed it.  Making it an explicit build requirement will reduce the amount of extra builds and deploys from changes to this simple script.

Reply:
I plan to use this script in polyphemus, too, will this change affect that? Will I have to inject that script in the Polyphemus Dockerfile?

Reply:
This script is now built into etna-base, which polyphemus is built on, so it will still have this script.  The change is merely moving that explicitly into that dockerfile as opposed to "magic injection" in the build_image script.